app/controllers/hub/outbound_calls_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Hub
+  class OutboundCallsController < ApplicationController
+    include AccessControllable
+    before_action :require_sign_in, except: :call
+    skip_before_action :verify_authenticity_token, only: :call
+    load_and_authorize_resource :client, except: :call
+
+    layout "admin"
+
+    def create
+      @form = OutboundCallForm.new(permitted_params, client: @client, user: current_user)
+      call = @form.call!
+      render :new and return unless call.present?
+      
+      redirect_to hub_client_outbound_call_path(client_id: @client.id, id: call.id)
+    end
+
+    def show
+      @outbound_call = OutboundCall.find(params[:id])
+    end
+
+    def call
```

Comment:
Consider getting the OutboundCall ID # from Twilio, not the phone number itself.